src/main/src/test/java/org/geoserver/test/GeoServerAbstractTestSupport.java

```diff
@@ -1352,5 +1353,20 @@ public int readLine(byte[] b, int offset, int length) {
 
             return i;
         }
+
+        @Override
+        public boolean isFinished() {
+            return false;
+        }
+
+        @Override
+        public boolean isReady() {
+            return false;
+        }
```

Comment:
See other comments, could be coded correctly with little effort.

Reply:
Done